.github/workflows/ci_pipe.yml

```diff
@@ -21,8 +21,16 @@ on:
       run_check:
         required: true
         type: boolean
+      conda_core_run_build:
```

Comment:
I'm thinking in most cases if we're building conda packages in CI we would want to build all of them.

Reply:
I agree with that and I was tempted to use the same label but it also seemed useful to create one for "core" as that is likely going to be the code that is updated most often.

I will revisit this when the next conda package is created. There may not be much value in proliferating labels.